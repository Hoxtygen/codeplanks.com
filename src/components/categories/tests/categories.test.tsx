import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import { Categories } from "../..";

describe("Categories", () => {
  it("should render the Categories component", () => {
    const { container } = render(<Categories />);
    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
