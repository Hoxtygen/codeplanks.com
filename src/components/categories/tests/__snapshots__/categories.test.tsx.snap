// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories should render the Categories component 1`] = `
<div>
  <div
    class="bg-white shadow-lg rounded-lg p-8 mb-8 pb-12"
  >
    <h3
      class="text-2xl mb-4 font-semibold border-b pb-4"
    >
      Categories
    </h3>
  </div>
</div>
`;
